### ISSUE TITLE
[BUG] CyberPanel attempts to renew SSL for Suspended domains

### ISSUE THREAD

**Issue by hbrawdy** [litespeedtech/cyberpanel#1172]
The update script renew.py attempts to renew all SSL certificates for all websites on the server, even if the websites have been suspended.  This forced a delay and a self-signed certificate, as well as a failure email to root.

**To Reproduce**
1. create a website ( MYTESTSITE.COM )
2. generate LetsEncrypt SSL certificate for website
3. suspend website
4. on renewal, LetsEncrypt fails, CyberPanel generates self-signed SSL

**Expected behavior**
renew.py should only loop through websites that are not suspended.

**Operating system:**
Centos 7

**CyberPanel version:**
Current Version:  2.3
Build:  4
Current Commit:  c8da443334b8815ec250a0fba606e75355f641cd

**Comment by cagivacode** [comment]
I have to agree with this, I have a test site (a sub domain) that I use to demo web designs.
I keep it suspended when I have no projects in the pipeline, and when the Cert expires there is an endless retry, which of course fails because the website itself is suspended.
I have resorted to unsuspending the test site every couple of months to get around this.

**Comment by master3395** [comment]
Thanks for the report. This issue is older and may not apply to current releases.

Please confirm:
1) CyberPanel version + build (please retest on the latest public release **2.4.9**)
2) OS name and version (whatever you run, e.g. `cat /etc/os-release`)
3) Exact steps still failing on a current install
4) Relevant logs (no passwords)

Note: the latest public release is **2.4.9**. Development branches are not recommended for reproducing bugs.

If we do not hear back within 30 days, this issue may be closed as stale. You can reopen with fresh details anytime.
